docs/examples/hook.sh

```diff
@@ -74,4 +74,18 @@ function unchanged_cert {
     #   The path of the file containing the intermediate certificate(s).
 }
 
+function invalid_challenge {
```

Comment:
hook.sh is in /bin/sh not in bash. (function don't exist)
invalid_challenge() {